wagtail/admin/panels/inline_panel.py

```diff
@@ -176,5 +203,25 @@ def get_comparison(self):
 
         def get_context_data(self, parent_context=None):
             context = super().get_context_data(parent_context)
-            context["can_order"] = self.formset.can_order
+            can_order = self.formset.can_order
+            context["can_order"] = can_order
+
+            attrs = context["attrs"]
+            # Set data-controller, including any existing value
+            # Used by the w-formset controller (FormsetController)
+            # to add dynamic behaviour
+            attrs["data-controller"] = " ".join(
```

Comment:
We will want to add unit tests to ensure we do correctly keep any `data-controller` that's set on an `InlinePanel` panel class.